html-css/building-blocks/step2-instructions.md

```diff
@@ -21,12 +21,14 @@ Click the <strong>Copy to Editor</strong> button below to add html for a chip co
 &lt;/div&gt;
 </pre>
 
-2) <strong>Add the badge inside of the chip. </strong> To do this add this block of code between `pf-c-chip__text` and `pf-c-button`.
+2) <strong>Add the `badge` inside of the `chip`. </strong>
+
+To do this add this block of code between `pf-c-chip__text` and `pf-c-button`.
 
 ```
 <span class="pf-c-badge pf-m-read">
   7
 </span>
 ```
 
-<strong>Pro Tip:</strong> Remember this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip it receives styling that gives it a margin. You can read more in the documentation [here](https://www.patternfly.org/v4/documentation/core/components/chip)
+<strong>Note: </strong> Remember this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip it receives styling that gives it a margin. You can read more in the documentation [here](https://www.patternfly.org/v4/documentation/core/components/chip)
```

Comment:
```suggestion
<strong>Note: </strong> Remember this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip, it receives styling that gives it a margin. You can read more in the documentation [here](https://www.patternfly.org/v4/documentation/core/components/chip)
```